libs/sql-schema-describer/src/mysql.rs

```diff
@@ -12,6 +12,15 @@ use std::{
 };
 use tracing::trace;
 
+/// Matches a default value in the schema, wrapped single quotes.
+///
+/// Example:
+///
+/// ```ignore
+/// 'this is a test'
+/// ```
+static DEFAULT_QUOTES: Lazy<Regex> = Lazy::new(|| Regex::new(r"'([\S\s]*)'").unwrap());
```

Comment:
Should we match anywhere in the default, or only the outermost characters? (I have a feeling `^'(.*)'$` may be more robust, and for sure faster)